### DISCUSSION TITLE
Add Swiper Js in argento theme and need responsive controls with same

### DISCUSSION THREAD

**Discussion by JamesDenish** [breezefront/community#213] [1. Help]
Hello @vovayatsyuk 

I need to integrate Swiper js into my argento theme used by one of our client I have done r&d regarding responsive controls but I am not able to get perfect result in responsive screens.

![image](https://github.com/breezefront/community/assets/142228975/a2598397-9017-4621-8568-b438519bea2b)
Above is the desktop view and below is the responsive mobile view.

Kindly help me out as much as possible it's urgent for one of our client.

Thanks in advance.....

**Comment by JamesDenish** [comment]
![tera](https://github.com/breezefront/community/assets/142228975/459e93d0-a96a-445e-b31e-aa812f9950c1)
This above screenshot is for responsive screen. Kindly help us.

@vovayatsyuk

**Comment by vovayatsyuk** [comment]
It looks like some issues with css/media queries. I don't know if it's related to Breeze. I recommend using the Chrome developer toolbar to identify and fix the issue.

**Comment by JamesDenish** [comment]
Hello @vovayatsyuk 

I want to manage number of slides as per media query and I am not quite sure how to manage in swiper js.

Please reply it's urgent.

Thanks in advance.

**Comment by JamesDenish** [comment]
@vovayatsyuk 
$('#owlsliderCat').owlCarousel({
                    items: 10,
                    autoplay: true,
                    autoplayHoverPause: true,
                    nav: true,
                    dots: true,
                    lazyLoad: true,
                    navText: ["<i class='pe-7s-angle-left'></i>","<i class='pe-7s-angle-right'></i>"],
                    responsive:{
                        0:{
                            items:1
                        },
                        768:{
                            items:2
                        },
                        992:{
                            items:5
                        }
                    }
                });
            });
     Like the above one.
     Thanks

**Reply by vovayatsyuk** [reply]
Swiper is not the part of Breeze. Do you use https://swiperjs.com/? Have you tried this: https://swiperjs.com/swiper-api#param-breakpoints?

**Comment by JamesDenish** [comment]
@vovayatsyuk 
I have used the js which you have provided but where to add the breakpoints function is not sure for me.

Can you help me out.

Thank You

**Reply by vovayatsyuk** [reply]
> I have used the js which you have provided

What js? I don't see any.

p.s. Do you mean the docs link? It's just the docs, that you need to properly incorporate into your code.

**Comment by JamesDenish** [comment]
@vovayatsyuk 
![image](https://github.com/breezefront/community/assets/142228975/cc7b6b62-83fd-489c-bf80-e1a997f5f9c3)

https://development.aqua-terra24.de/en/

You can refer above link to inspect swiper-slider

**Reply by vovayatsyuk** [reply]
Sorry, this question is not about Breeze. As I understand you are using the `swissup/highlight` module and trying to modify slide count per screen size. This module doesn't provide such feature.

**Comment by JamesDenish** [comment]
@vovayatsyuk 

But whom to ask for Swissup/Highlight module to implement the current question.

Please do let me know.
Thanks in advance

**Reply by vovayatsyuk** [reply]
You can contact swissuplabs or argento support. It depends on where you've made a purchase.
